.github/workflows/create-azure-self-hosted-runners.yml

```diff
@@ -40,6 +40,8 @@ env:
   # Note that you'll need "p" (arm64 processor) and ideally "d" (local temp disk). The number 8 stands for 8 CPU-cores.
   # For a convenient overview of all arm64 VM types, see e.g. https://azureprice.net/?_cpuArchitecture=Arm64
   AZURE_VM_TYPE: Standard_D8plds_v5
+  # At the time of writing, "centralindia" was the cheapest region for the VM type we're using. https://azureprice.net/vm/Standard_D8plds_v5
+  AZURE_VM_REGION: centralindia
```

Comment:
The commit message says `westindia`, while `centralindia` seems to be correct.

Reply:
Ugh 🤦🏼 thanks for being so sharp on this! My brain is not fully awake yet. Will fix